webpack.config.js

```diff
@@ -6,12 +6,13 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const ASSET_PATH = process.env.ASSET_PATH || '/';
 
 module.exports = {
-  entry: './src/index.js',
+  entry: ['@babel/polyfill', './src/index.js'] ,
```

Comment:
what does this do?

Reply:
Because sagas using generators that still not implemented in all browsers. So we using polyfills for that.